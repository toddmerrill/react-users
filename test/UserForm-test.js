import React from 'react';
import { UserForm } from '../src/components/Users.js';
import renderer from 'react-test-renderer';
import mock from './user-mocks'

it('UserForm renders correctly', () => {
    const tree = renderer.create(
        <UserForm user={mock.user} onBlur={mock.f} onChange={mock.f}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
