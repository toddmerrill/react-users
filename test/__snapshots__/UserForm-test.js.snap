// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm renders correctly 1`] = `
<div
  className="rightList"
>
  <h2>
    Personal Information
  </h2>
  <form
    className="form-horizontal"
  >
    <div
      className="form-group"
    >
      <label
        className="controlLabel"
        htmlFor="inputfirstName"
      >
        First Name
      </label>
      <div
        className="controls"
      >
        <input
          autoFocus={true}
          className="form-control"
          id="inputfirstName"
          name="firstName"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="First Name"
          tabIndex={1}
          type="text"
          value="test-first"
        />
      </div>
    </div>
    <div
      className="form-group"
    >
      <label
        className="controlLabel"
        htmlFor="inputlastName"
      >
        Last Name
      </label>
      <div
        className="controls"
      >
        <input
          autoFocus={undefined}
          className="form-control"
          id="inputlastName"
          name="lastName"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Last Name"
          tabIndex={2}
          type="text"
          value="test-last"
        />
      </div>
    </div>
    <div
      className="form-group"
    >
      <label
        className="controlLabel"
        htmlFor="inputage"
      >
        Age
      </label>
      <div
        className="controls"
      >
        <input
          autoFocus={undefined}
          className="form-control"
          id="inputage"
          name="age"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Age"
          tabIndex={3}
          type="text"
          value="987"
        />
      </div>
    </div>
  </form>
</div>
`;
